src/sidebar/components/hypothesis-app.js

```diff
@@ -73,18 +72,10 @@ function HypothesisAppController(
   // Reload the view when the user switches accounts
   $scope.$on(events.USER_CHANGED, function (event, data) {
     self.auth = authStateFromProfile(data.profile);
-    self.accountDialog.visible = false;
   });
 
-  session.load().then(function (profile) {
-    // When the authentication status of the user is known,
-    // update the auth info in the top bar and show the login form
-    // after first install of the extension.
+  session.load().then(profile => {
     self.auth = authStateFromProfile(profile);
-
-    if (!profile.userid && settings.openLoginForm && !auth.login) {
-      self.login();
-    }
```

Comment:
Do we need to remove the `openLoginForm` setting? From the docs, and various parts of the code that read it and pass it from host page into sidebar.

Reply:
Done. I do wonder whether we should keep a list of obsolete/removed settings around anywhere for future reference, and in general we need to be careful about removing settings. In this case however I'd be quite surprised if anyone else was using it since we added it explicitly for use on the onboarding page.